src/api/interface/index.ts

```diff
@@ -49,6 +42,37 @@ export interface Permission {
   name: string;
 }
 
+export interface TestInterpretationDTO {
+  namePattern: string;
```

Comment:
I don't quite understand the significance of `namePattern` and `variables`, can you elaborate on how they're used?

Also: I feel like `theme` should rather be something like `sentiment`, with defined enum values

Reply:
**On theme:**
Theme is already an enum of currently 4 possible values:  https://github.com/cov-clear/backend/blob/master/src/domain/model/test/interpretation/Interpretation.ts#L5

**On namePattern**
It was done to allow easy localisation. 
```
[{
    namePattern: "PCR score {{score}}"
    theme: "NEUTRAL"
    variables: {
        score: 39
    }
}]
```

In fact, if we start doing localisation maybe it would make sense to do this on the server-side 🤔 

This would simplify the Interpretation for the client. I'll update 👍